### ISSUE TITLE
Support for asset´s custom propterties are missing for az iot ops asset create

### ISSUE THREAD

**Issue by bindsi** [Azure/azure-iot-ops-cli-extension#203]
**Describe the bug**
Azure IoT Operations portal recently introduced custom properties for assets apart from the eight existing preconfigured manufacturer, url, software-revision, hardware-revision etc. ones.
While the 8 preconfigured ones are available as parameters, a parameter collection for the generic custom properties is still missing for the `az iot ops asset create` [command](https://learn.microsoft.com/en-us/cli/azure/iot/ops/asset?view=azure-cli-latest#az-iot-ops-asset-create)

**To Reproduce**
facilitate `az iot ops asset create` command --> no parameter for custom properties available

**Expected behavior**
custom-properties collection parameter

**Screenshots**
<img width="1090" alt="image" src="https://github.com/Azure/azure-iot-ops-cli-extension/assets/16034499/1a73745d-0bfe-4eb8-9969-0ebb2b13602a">

**Comment by vilit1** [comment]
Thanks for letting us know! We will include custom attributes in the next release.

**Comment by bindsi** [comment]
Awesome, thanks!!! Should we leave this issue open for tracking?

**Comment by digimaun** [comment]
Yea let's leave this issue open until the update gets merged & released.

**Comment by digimaun** [comment]
v0.5.0b2 is released with requested functionality via `--attr` / `--custom-attribute`. Example usage in `az iot ops asset create --help`. Let us know if you run into any issues, and we can re-open.

**Comment by bindsi** [comment]
Awesome, thanks for your support...we will try this out and let you know if we face any issue.

**Comment by danuw** [comment]
Hi @bindsi did you manage to use `--attr` to add customer properties in the end? I get personally get errors (`cluster <name> not found` )

**Comment by bindsi** [comment]
@danuw sorry, I totally forgot to validate this, thanks for reminding me!

I just executed the following command and it worked like a charm:
az iot ops asset create --name dapr -g rg-dapr-workflow --cluster arc-dapr-workflow --endpoint opc-ua-broker-opcplc-000000-azure-iot-operatio --attr customProp='customValue' --data data_source='ns=3;s=FastUInt10' name=temperature
<img width="1106" alt="image" src="https://github.com/Azure/azure-iot-ops-cli-extension/assets/16034499/17963a6c-528c-4bb9-89a2-fd2f002428cc">

Have you updated your az cli extension azure-iot-ops?
run `az extension update --name azure-iot-ops` and retry!!!

cc: @digimaun

**Comment by danuw** [comment]
Hi thanks, I did check I was on the latest and checked in the release notes which version this came with (which is how I found this page).
It did not work for me using the notation you just showed (I tried on codespace and I tried on Ubuntu on a local compact pc style device)
Let me try again one last time to be extra sure when I am back at my desk.

**Comment by digimaun** [comment]
@danuw assuming you are on the latest, from this [prior msg](https://github.com/Azure/azure-iot-ops-cli-extension/issues/203#issuecomment-2217204983) you mentioned getting an error about your cluster not being found. Please double-check you are using the right subscription you expect the cluster to be in.
